ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/base/v1/base/clientlibs/editor/js/editDialog.js

```diff
@@ -122,6 +122,22 @@
         }
     }
 
+    function validateName() {
```

Comment:
We are not creating any jcr bode with the name option. The validation should align with the AF2 spec rather than the jcr name validation

Reply:
https://jira.corp.adobe.com/browse/CQ-4349511
As per @vdua in the above ticket as per AF2 spec only restriction on name field is that it shouldn't begin with `$`.
@nit23uec I hope this is what you are trying to say here..

Reply:
The problem with that is the name will not work in DoR as per the AF Spec currently. We need to fix that in DoR before removing this restriction. We have to make that work before removing this restriction.

Reply:
Also the rule editor would need some changes as well while transforming special names.